src/Files.App/Data/Items/NavigationBarSuggestionItem.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Data.Items
+{
+	public class NavigationBarSuggestionItem : ObservableObject
+	{
+		private string? _Text;
+		public string? Text
+		{
+			get => _Text;
+			set => SetProperty(ref _Text, value);
+		}
+
+		private string? _PrimaryDisplay;
+		public string? PrimaryDisplay
```

Comment:
can you describe more? like header, description, hotkey, path. or you concern about we might use this in different purpose? i dont think we need to think of it.

Reply:
> you concern that we might use this in different purpose?

Yes, I also use this to suggest paths. Right now I only use PrimaryDisplay for path suggestions, but it can be extended to display both localized folder names and path, for example.

Reply:
Can you add xml commes describing the purpose(like, this is used for command palette item header) for each?